interface/patient_file/summary/stats_full.php

```diff
@@ -202,220 +181,252 @@ function newEncounter() {
-                                    $click_class = '';
-                                } elseif ($row['erx_uploaded'] == 1 && $focustype == 'medication') {
-                                    $click_class = '';
-                                }
+                            $shortBegDate = text(trim(oeFormatShortDate($row['begdate'])));
+                            $shortEndDate = text(trim(oeFormatShortDate($row['enddate'])));
+                            $fullBegDate = text(trim(oeFormatDateTime($row['begdate'])));
+                            $fullEndDate = text(trim(oeFormatDateTime($row['enddate'])));
+                            $shortModDate = text(trim(oeFormatShortDate($row['modifydate'])));
+                            $fullModDate = text(trim(oeFormatDateTime($row['modifydate'])));
 
-                                echo " <tr class='" . attr($bgclass) . " detail' $colorstyle>\n";
-                                if ($canSelect) {
-                                    echo "  <td><input type='checkbox' class='selection-check' id='sel_" . attr($rowid) . "'
-                                        onclick='rowSelectionChanged(" . attr_js($focustype) . ");'/></td>\n";
-                                }
-                                echo "  <td class='text-left " . attr($click_class) . "' style='text-decoration: underline' id='" . attr($rowid) . "'>" . text($disptitle) . "</td>\n";
-                                echo "  <td>" . text(trim(oeFormatDateTime($row['begdate']))) . "&nbsp;</td>\n";
-                                echo "  <td>" . text(trim(oeFormatDateTime($row['enddate']))) . "&nbsp;</td>\n";
-                                // both codetext and statusCompute have already been escaped above with htmlspecialchars)
-                                echo "  <td>" . $codetext . "</td>\n";
-                                echo "  <td>" . $statusCompute . "&nbsp;</td>\n";
-                                echo "  <td class='nowrap'>";
-                                echo generate_display_field(array('data_type' => '1','list_id' => 'occurrence'), $row['occurrence']);
-                                echo "</td>\n";
-                                if ($focustype == "allergy") {
-                                    echo "  <td>";
+                            $outcome = ($row['outcome']) ?  generate_display_field(['data_type' => 1, 'list_id' => 'outcome'], $row['outcome']) : false;
+                            ?>
+                        <div class="list-group-item p-1">
+                            <div class="summary m-0 p-0 d-flex w-100 justify-content-end align-content-center">
+                                <?php if ($canSelect) : ?>
+                                    <input type="checkbox" class="selection-check mt-1 mr-2" data-issue="<?php echo xla($t); ?>" name="sel_<?php echo attr($rowid); ?>" id="sel_<?php echo attr($rowid); ?>">
```

Comment:
per above, should data-issue be translated. And if it should would it break if they all got translated to "dummy"